DOCS-7400 Added Use Cases to Kafka (#725)
mulesoft/docs-connectors: modules/ROOT/pages/kafka/kafka-connector.adoc

Kafka Connector v4.0

Anypoint Connector for Apache Kafka (Apache Kafka Connector) enables you to interact with the Apache Kafka messaging system and achieve seamless integration between your Mule app and an Apache Kafka cluster, using Mule runtime engine (Mule).

For compatibility information see the xref:release-notes::connector/kafka-connector-release-notes-mule-4.adoc[Apache Kafka Connector Release Notes].

== Prerequisites

Before creating an app, you must:

* Have access to the Apache Kafka target resource and Anypoint Platform
* Understand how to create a Mule app using Design Center or Anypoint Studio

> * Have access to Apache Kafka to get values for the fields that appear in Studio and Design Center

Reviewer: ```suggestion
* Have access to Apache Kafka to get values for the fields that appear in Anypoint Studio and Design Center
```

== Use Cases

> Apache Kafka is a multi-purpose application. Integration with Apache Kafka relies on the applications that are typically used with it. 

Reviewer: ```suggestion
Apache Kafka is a multipurpose application. Integration with Apache Kafka relies on the applications that are typically used with it. 
```

> A few examples of use cases include:

Reviewer: ```suggestion
Example use cases include:
```

> * An integration application that ensures patients in a hospital receive the care they need in a timely manner +

Reviewer: ```suggestion
* An integration application that ensures that patients in a hospital receive the care they need in a timely manner +
```

> An example of this is a hospital server that processes emergency requests for patient admissions. The requests are ordered by priority, which is determined by specified criteria, such as how critical the patient's condition is and the staff that is available to treat the patient. In this scenario, an application processes Apache Kafka messages in the order that they are received, relying on the order and idempotency of the messages sent through the queue. 

Reviewer: ```suggestion
An example of this is a hospital server that processes emergency requests for patient admissions. The requests are ordered by priority determined by specified criteria, such as how critical the patient's condition is and the staff that is available to treat the patient. In this scenario, an application processes Apache Kafka messages in the order that they are received, relying on the order and idempotency of the messages sent through the queue. 
```

* An application that is time-sensitive +

> An example is a newsroom that uses the Apache Kafka system to deliver the latest news. The news must always arrive at the latest point first. In this case, reading from the Apache Kafka queue sometimes requires skipping to the end of the queue in order to retrieve the latest news first. 

Reviewer: ```suggestion
An example is a newsroom that uses the Apache Kafka system to deliver the latest news. To retrieve the latest news, reading from the Apache Kafka queue sometimes requires reading from the end of the queue first. 
```

> * Multiple applications that are reliant on the information provided by Kafka +

Reviewer: ```suggestion
* Multiple applications that rely on the information provided by Kafka +
```
